packages/host/app/services/command-service.ts

```diff
@@ -62,3 +90,13 @@ function hasPatchData(payload: any): payload is PatchPayload {
       'relationships' in payload)
   );
 }
+
+function hasSearchData(payload: any): payload is SearchPayload {
+  return (
+    typeof payload === 'object' &&
+    payload !== null &&
+    'card_id' in payload &&
+    'filter' in payload &&
```

Comment:
should we be using something like this for a more exhaustive check on the type: https://github.com/cardstack/boxel/blob/main/packages/runtime-common/query.ts#L214